reactioncommerce/reaction
# Fix on Console Errors when clicking order search results

**Fixes #2569** 

**Testing**
- Login as admin
- Enable shipping and Stripe
- Place an order
- Go to home screen
- Go to Search and select orders
- Enter a search query that will bring up results (such as your email)
- Click on the results
- Observe that no error shows up in the console
